public/docs/ts/latest/guide/structural-directives.jade

```diff
@@ -163,7 +166,7 @@ figure.image-display
   We can confirm these effects by wrapping the middle "hip" of the phrase "Hip! Hip! Hooray!" within a `<template>` tag.
 +makeExample('structural-directives/ts/app/structural-directives.component.html', 'template-tag')(format=".")
 :marked
-  The display is a 'Hip!' short of perfect enthusiasm. The DOM effects are different when Angular is control.
+  The display is a 'Hip! Hooray!', short of perfect enthusiasm. The DOM effects are different when Angular is control.
```

Comment:
Think there is an issue with end of the sentence. Do we mean: 'when Angular IS in control'?